tests/eosio.finalizer_key_tests.cpp

```diff
@@ -564,14 +490,6 @@ BOOST_FIXTURE_TEST_CASE(update_elected_producers_finalizers_replaced_test, final
    // head_finality_data is available when nodoes is in Savanna
    BOOST_REQUIRE_EQUAL( true, control->head_finality_data().has_value() );
 
-   // Verify last finalizer key id table contains all finalzer keys
```

Comment:
These types of checks should still be possible by instead looking up the table that contains the last proposed finalizer keys.